js/metroInit.js

```diff
@@ -33,62 +33,77 @@ UIFreewallVertical = (function() {
 
   UIFreewallVertical.setupFreewalVertical = function(data) {
     var wall = new freewall('[data-metro-id="' + data.id + '"]:not([data-mce-bogus] [data-metro-id="' + data.id + '"])');
-    var cellWidth;
-    var cellHeight;
+
+    function resizePanels() {
+      var containerWidth = $('[data-metro-id="' + data.id + '"]:not([data-mce-bogus] [data-metro-id="' + data.id + '"])').parent().width();
+      var $panels = $('.panels.size11, .panels.size21, .panels.size22');
+      var currentClasses = $panels.attr('class');
+      var indexOfContainerClass = currentClasses.indexOf('panel-container');
+
+      if (indexOfContainerClass > -1) {
+        var styleToRemove = currentClasses.substring(indexOfContainerClass);
+        $panels.removeClass(styleToRemove);
```

Comment:
What classes are you trying to remove here? Where do they come from?

Also, if you're trying to remove all the classes:

1. Change `styleToRemove` to `classesToRemove` because you're removing classes, not styles.
1. Use `Element.prototype.classList` and loop through the classes to remove all the classes except for `panel-container`. It's more specific, accurate and long term proof this way.

Reply:
We need to remove the previous class that we assign (for example panel-container-400)

Reply:
@MaksymShokin I see. In this case, let's not use classes because it's a cumbersome way to toggle the classes.

Change the method slightly to use a `data-container-size="400"` and so on instead of a class. The selectors for CSS can then use `[data-container-size="400"]` or `[data-container-size]` etc.

Reply:
@tonytlwu 
Actually, when we switched to assigning classes to container it became pretty straightforward. 
![classes](https://user-images.githubusercontent.com/52824207/81780511-50b79000-94ff-11ea-9615-874d1ed0d348.PNG).
We just remove previous class and assign new one. Do you think we still need to switch to data attributes in this case?

Reply:
@MaksymShokin Yes I’d say still change the method as I suggested. It takes very little effort and risk now, and ensures that the entire class list isn’t occupied for this usage.

Reply:
@tonytlwu 
Fixed

Reply:
@MaksymShokin Good job. Looks clean and easily understandable.